package.json

```diff
@@ -66,7 +67,7 @@
     "js-yaml": "^3.13.1",
     "keycloak-connect": "^7.0.1",
     "next": "^9.3.1",
-    "node-fetch": "^2.6.0",
+    "node-fetch": "^3.0.0-beta.4",
```

Comment:
Are we completely switched over at this point to axios?  Do we still need this dependency?

Reply:
I will double check uploads one last time and then I can pull the plug on `node-fetch`